src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -345,8 +345,8 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     /**
      * Given an existing path, convert it to a path relative to a given starting path.
      *
-     * @param string $endPath   Absolute path of target
-     * @param string $startPath Absolute path where traversal begins
+     * @param string $endPath   Path of target
+     * @param string $startPath Path where traversal begins
```

Comment:
It should support relative paths for some reasons:

1. It worked before 3.2.7!
2. Absolute paths will still work (No BC break).
3. As @Loilo mentioned, the documentation doesn't tell that only absolute paths can be used.
4. It makes life easier (Otherwise everyone have to do extra checks on absolute paths, before using this method).